src/crypto/CHIPCryptoPAL.h

```diff
@@ -492,6 +492,20 @@ class P256Keypair : public P256KeypairBase
      **/
     CHIP_ERROR Initialize(ECPKeyTarget key_target) override;
 
+    /**
+     * @brief Import a raw key pair (bytes) into the keypair
+     * @return Returns a CHIP_ERROR on error, CHIP_NO_ERROR otherwise
+     **/
+    CHIP_ERROR ImportRawKeypair(const uint8_t * key_data, size_t key_data_size);
```

Comment:
Where would this sort of key data come from?  How does one actually use this API?